File: CS/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...

namespace ChartAppearance {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // Create an empty chart.
            ChartControl barChart = new ChartControl();

            // Create a series and add points to it.
            Series series1 = new Series("Bar Series", ViewType.Bar);
            series1.Points.Add(new SeriesPoint("A", new double[] { 10 }));
            series1.Points.Add(new SeriesPoint("B", new double[] { 12 }));
            series1.Points.Add(new SeriesPoint("C", new double[] { 14 }));
            series1.Points.Add(new SeriesPoint("D", new double[] { 17 }));

            // Add the series to the chart.
            barChart.Series.Add(series1);

            // Customize the chart's appearance.
            barChart.AppearanceName = "Dark";
            barChart.PaletteName = "Opulent";
            barChart.PaletteBaseColorNumber = 5;

            // Add the chart to the form.
            barChart.Dock = DockStyle.Fill;
            this.Controls.Add(barChart);
        }

    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...

Namespace ChartAppearance
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
            ' Create an empty chart.
            Dim barChart As New ChartControl()

            ' Create a series and add points to it.
            Dim series1 As New Series("Bar Series", ViewType.Bar)
            series1.Points.Add(New SeriesPoint("A", New Double() { 10 }))
            series1.Points.Add(New SeriesPoint("B", New Double() { 12 }))
            series1.Points.Add(New SeriesPoint("C", New Double() { 14 }))
            series1.Points.Add(New SeriesPoint("D", New Double() { 17 }))

            ' Add the series to the chart.
            barChart.Series.Add(series1)

            ' Customize the chart's appearance.
            barChart.AppearanceName = "Dark"
            barChart.PaletteName = "Opulent"
            barChart.PaletteBaseColorNumber = 5

            ' Add the chart to the form.
            barChart.Dock = DockStyle.Fill
            Me.Controls.Add(barChart)
        End Sub

    End Class
End Namespace